pkgs/development/python-modules/angr/default.nix

```diff
@@ -87,7 +95,7 @@ buildPythonPackage rec {
     sortedcontainers
     sqlalchemy
     sympy
-    unicorn
+    unicorn'
```

Comment:
It looks like we should now have the ability to depend on `unicorn-angr` (even if just temporarily):
https://github.com/NixOS/nixpkgs/pull/388155